backend/app/views/spree/admin/shared/preference_fields/_encrypted_string.html.erb

```diff
@@ -0,0 +1,12 @@
+<% label = local_assigns[:label].presence %>
+<% html_options = {class: 'input_string fullwidth'}.update(local_assigns[:html_options] || {}) %>
```

Comment:
`Hash#update` just does the same thing as `Hash#merge` and we use `merge` a ton in the codebase already so I think we should prefer `merge` for consistency.

Reply:
Hi @jarednorman, thanks for the feedback. I've looked at the other preference_fields partial on the backend and all of them are using `Hash#update` instead of `Hash#merge`. Probably we should change the method to `Hash#merge` for all the partials to be more consistent with the overall usage of the method inside the codebase, but probably I would prefer to do tath in another PR, what do you think about it?

Thanks again for your feedback 😄.

Reply:
Interesting, I had not realized we had consistently used that in the other partials, though I tend to think we should still prefer `merge` as we use it pretty heavily in other parts of the codebase.

Reply:
I don't have a strong opinion about it, so I've switched from `update` to `merge`